docs_zh-CN/get_started.md

```diff
@@ -1 +1,208 @@
 ## 依赖
+
+- Linux or macOS (Windows下支持需要mmcv-full，但运行时可能会有一些问题。)
+- Python 3.6+
+- PyTorch 1.3+
+- CUDA 9.2+ (If you build PyTorch from source, CUDA 9.0 is also compatible)
+- GCC 5+
+- [MMCV](https://mmcv.readthedocs.io/en/latest/#installation)
+
+可编译的MMSegmentation和MMCV版本如下所示，请对照对应版本安装以避免安装问题。
```

Comment:
Please check if there is a space between Chinese and English in this doc.

Reply:
> Please check if there is a space between Chinese and English in this doc.

There must have a blank line between line 8&10, otherwise error will raise.

Reply:
A space between Chinese and English is needed,
e.g. 可编译的MMSegmentation和MMCV版本如下所示 --> 可编译的 MMSegmentation 和 MMCV 版本如下所示.
Please check these cases in all docs.

Reply:
> A space between Chinese and English is needed,
> e.g. 可编译的MMSegmentation和MMCV版本如下所示 --> 可编译的 MMSegmentation 和 MMCV 版本如下所示.
> Please check these cases in all docs.

Done.